eCourse_backend/eCourse_backend/static/readme.md

```diff
@@ -0,0 +1,12 @@
+# Readme
+
+* **/Css:**
+    * Folder contains css files that are served as static files
+* **./:**
+    * Contains:
+        * different Versions of the Logo + some other files
+            * used to create compability with different browser features (some are just for Android/Windows/...)
+            * other files are also needed for these features (e.g. site.webmanifest)
+            * created with: [RealFaviconGenerator](https://realfavicongenerator.net/)
+        * the original logo
+            * created with: [FreeLogoDesign](https://de.freelogodesign.org)
```

Comment:
At the moment it doesn't hurt and otherwise it would be quite confusing why these files are there
I would leave it there since djangos static file hosting shouldn't be used in a production environment anyways